File: asp.net/sdk-lpl-mvc5/sdk-lpl-mvc5/Models/UserInfoModel.cs
using System;

namespace sdk_lpl_mvc5.Models
{
    /// <summary>
    /// Modèle de réponse pour la vérification de compte
    /// Représente le modèle de données envoyé en réponse d'une requête de vérification de compte partenaire
    /// </summary>
    public class UserInfoModel
    {
        public int PartenaireID { get; set; }
        public string CodeUtilisateur { get; set; }
        public string Mail { get; set; }
        public string TypeAbonnement { get; set; }
        public DateTime? DateExpiration { get; set; }
        public DateTime? DateSouscription { get; set; }
        public bool AccountExist { get; set; }

        public void CreateDummyModel()
        {
            model.AccountExist = true;
            model.Mail = "dummy@gmail.com";
            model.CodeUtilisateur = "dummy1234";
        }
    }
}

File: SDKPHP/models/UserInfoModel.php
<?php
/**
* Modèle de réponse pour la vérification de compte
* Représente le modèle de données envoyé en réponse d'une requête de vérification de compte partenaire
* @author  Bastien Caubet <bastien@nextinpact.com>, Luc Raymond <luc@nextinpact.com>
*/
class UserInfoModel
{
    public $PartenaireID;
    public $Mail;
    public $CodeUtilisateur;
    public $TypeAbonnement;
    public $DateExpiration;
    public $DateSouscription;
    public $AccountExist;


    public function CreateDummyModel() {
        $this->Mail = "dummy@gmail.com";
        $this->CodeUtilisateur = "dummy";
        $this->AccountExist = TRUE;
    }
}
